python/lsst/meas/modelfit/mixture.cc

```diff
@@ -0,0 +1,217 @@
+// -*- lsst-c++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2008-2013 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include <sstream>  // Python.h must come before even system headers
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/eigen.h"
+
+#include "lsst/utils/pybind11.h"
+#include "lsst/meas/modelfit/Mixture.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace modelfit {
+namespace {
+
+using PyMixtureComponent = py::class_<MixtureComponent>;
+using PyMixtureUpdateRestriction = py::class_<MixtureUpdateRestriction,
+                                              std::shared_ptr<MixtureUpdateRestriction>>;
+using PyMixture = py::class_<Mixture, std::shared_ptr<Mixture>,
+                             afw::table::io::PersistableFacade<Mixture>, afw::table::io::Persistable>;
+
+PyMixtureComponent declareMixtureComponent(py::module & mod) {
```

Comment:
No, I was just uniformly forgetful on this ticket.  I'll go and make them all static.